modules/host.go

```diff
@@ -92,6 +92,12 @@ type (
 		ObligationStatus    uint64 `json:"obligationstatus"`
 	}
 
+	// HostWorkingState reports the working state of a host.
+	HostWorkingState string
+
+	// HostConnectabilityState reports the connectability state of a host.
+	HostConnectabilityState string
```

Comment:
These need to have well defined values that they can be. String is not enough, need to enumerate all possible results.